### DISCUSSION TITLE
kpanda/user-guide/storage/pv/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1715] [Announcements]
# kpanda/user-guide/storage/pv/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/storage/pv/

**Comment by liyuerich** [comment]
删除数据卷有什么前提和要求吗？

**Reply by lemontrees123** [reply]
建议先确定没有其他应用或服务依赖该存储卷后再删除。
